experimental/Schemes/IdealSheaves.jl

```diff
@@ -838,6 +838,7 @@ function maximal_associated_points(I::AbsIdealSheaf; covering=default_covering(s
     !is_one(I(U)) || continue                        ## supp(I) might not meet all components
     dim(I(U)) == dim(I) || continue                  ## components of lesser dimension can be discarded
     components_here = minimal_primes(I(U))
+    @assert all(p->is_prime(p), components_here)
```

Comment:
The minimal primes should be prime. Why do you need this assertion. Is there a problem with minimal_primes?